src/resources/views/includes/footer.blade.php

```diff
@@ -3,6 +3,7 @@
   <!-- To the right -->
   <div class="pull-right hidden-xs">
     <b>{{ trans('web::seat.render_in') }}</b> {{ number((microtime(true) - LARAVEL_START), 3) }}ms |
+    <b>{{ trans('web::seat.sde_version') }}</b> {{ Seat\Services\Settings\Seat::get('installed_sde') }}
```

Comment:
Use the [setting helper](https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L187) please.